package.json

```diff
@@ -10,18 +10,23 @@
     "ckeditor5-plugin"
   ],
   "dependencies": {
-    "@ckeditor/ckeditor5-core": "^12.0.0"
+    "@ckeditor/ckeditor5-core": "^12.0.0",
+    "@ckeditor/ckeditor5-ui": "^12.0.0",
+    "@ckeditor/ckeditor5-utils": "^12.0.0"
   },
   "devDependencies": {
     "@ckeditor/ckeditor5-basic-styles": "^11.0.0",
-    "@ckeditor/ckeditor5-clipboard": "^11.0.0",
```

Comment:
Typo.

Reply:
OMG I was reading this line like 20 times... I couldn't spot it. And I'm pretty sure that this typo is not in GH UI. The same with the IDE - it was highlighting a repo since it coulndn't match repo with NPM version. 

The culprit was a space at the end of the package name. Freaking space... I was looking at it and couldn't see it...